src/transformers/models/auto/modeling_tf_auto.py

```diff
@@ -439,6 +445,10 @@
 )
 
 TF_MODEL_MAPPING = _LazyAutoMapping(CONFIG_MAPPING_NAMES, TF_MODEL_MAPPING_NAMES)
+TF_MODEL_FOR_AUDIO_CLASSIFICATION_MAPPING_NAMES = _LazyAutoMapping(
```

Comment:
@adit299 The recursion is happening because of this line - the variable is being assigned to itself 

```suggestion
TF_MODEL_FOR_AUDIO_CLASSIFICATION_MAPPING = _LazyAutoMapping(
```

Reply:
Thanks, this solves the issue!